requirements-test.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.15.1
+numpy==1.16.1
```

Comment:
Fix the following error for Python 3.6:
```
ImportError while loading conftest '/home/travis/build/BayesWitnesses/m2cgen/tests/conftest.py'.
tests/conftest.py:3: in <module>
    from tests import utils
tests/utils.py:13: in <module>
    from lightning.classification import (
../../../virtualenv/python3.6.7/lib/python3.6/site-packages/lightning/classification.py:1: in <module>
    from .impl.adagrad import AdaGradClassifier
../../../virtualenv/python3.6.7/lib/python3.6/site-packages/lightning/impl/adagrad.py:10: in <module>
    from .base import BaseClassifier, BaseRegressor
../../../virtualenv/python3.6.7/lib/python3.6/site-packages/lightning/impl/base.py:14: in <module>
    from .randomkit import RandomState
../../../virtualenv/python3.6.7/lib/python3.6/site-packages/lightning/impl/randomkit/__init__.py:1: in <module>
    from .random_fast import RandomState
__init__.pxd:918: in init lightning.impl.randomkit.random_fast
    ???
E   ValueError: numpy.ufunc size changed, may indicate binary incompatibility. Expected 216 from C header, got 192 from PyObject
```

Refer to https://github.com/scikit-learn-contrib/hdbscan/issues/272#issuecomment-484383051.